src/quote/Quote.ts

```diff
@@ -87,6 +123,16 @@ export function fromQuoteDataAndIdentity(
   return createAttesterSignature(quoteInput, identity)
 }
 
+/**
+ * Creates a [[Quote]] signed by the Attester and the Claimer.
+ *
+ * @param claimerIdentity [[Identity]] of the Claimer in order to sign.
+ * @param attesterSignedQuote An Attester signed [[Quote]] object.
```

Comment:
Question:
Is the wording of "An attester signed" clear? Or should we say "... signed by an Attester"?

Reply:
Agreed